presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/thrift/MockThriftMetastoreClientFactory.java

```diff
@@ -37,7 +37,7 @@ public MockThriftMetastoreClientFactory(Optional<HostAndPort> socksProxy, Durati
     }
 
     @Override
-    public ThriftMetastoreClient create(HostAndPort address)
+    public ThriftMetastoreClient create(HostAndPort address, Optional<String> delegationToken)
```

Comment:
`checkArgument(!delegationToken.isPresent(), "delegation token is not supported")`